test/specs/core.interaction.tests.js

```diff
@@ -192,6 +192,47 @@ describe('Core.Interaction', function() {
 		});
 	});
 
+	describe('x-axis mode', function() {
```

Comment:
I would move this unit test in `global.deprecations.tests.js`, under version 2.4.0